spec/forms/steps/case/codefendants_form_spec.rb

```diff
@@ -1,22 +1,24 @@
 require 'rails_helper'
 
 RSpec.describe Steps::Case::CodefendantsForm do
-  let(:arguments) { {
-    crime_application: crime_application,
-    codefendants_attributes: codefendants_attributes,
-  } }
+  subject { described_class.new(arguments) }
+
+  let(:arguments) do
+    {
+      crime_application:,
+    codefendants_attributes:,
+    }
+  end
 
   let(:crime_application) { instance_double(CrimeApplication, case: case_record) }
   let(:case_record) { Case.new }
 
-  let(:codefendants_attributes) {
+  let(:codefendants_attributes) do
     {
-      "0"=>{"first_name"=>"John", "last_name"=>"Doe", "conflict_of_interest"=>"no"},
-      "1"=>{"first_name"=>"Jane", "last_name"=>"Doe", "conflict_of_interest"=>"yes"},
+      '0' => { 'first_name' => 'John', 'last_name' => 'Doe', 'conflict_of_interest' => 'no' },
```

Comment:
I think this change might have been caused by the cop that prefers single quotes for strings without interpolation on the basis that the single quoted string is more performant. I think it's been shown that there might not be a difference (eg this article https://www.viget.com/articles/just-use-double-quoted-ruby-strings/) and, if there is, it's pretty negligible, so I think we could safely turn that cop off.

Reply:
I'm personally happy to leave the cop on so that we end up with consistent quotes throughout